flow/connectors/clickhouse/normalize.go

```diff
@@ -305,6 +305,10 @@ func (c *ClickhouseConnector) NormalizeRecords(
 				if err != nil {
 					return nil, fmt.Errorf("error while converting column type to clickhouse type: %w", err)
 				}
+				if colType == qvalue.QValueKindNumeric {
```

Comment:
this can be if/else, since it disregards the above assignment to clickhouseType. Ideally `ToDWHColumnType` would take `column` & handle this